content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
 
-Split quotation: For example, Matthew 4:19 says, "Come, follow me", Jesus said, "and I will send you out to fish for people." This example could be written, Who said it? "Come, follow me, and I will send you out to fish for people" Answer: Jesus.
+The answer to the Situation question must be in context.
 
-### Question Structure
+##### Example
 
-This defines how a Quizmaster must introduce and read each question type.
+As an example split-quotation Situation question, consider Matthew 4:19:
 
-Four parts:
+> "Come, follow me," Jesus said, "and I will send you out to fish for people."
 
-1. Introduction part one
-    - "Question Number [ 1, 2, 3, 4, 5, etc ] is a [ TYPE ] question."
-2. Introduction part two
-    - For Situation questions only: "I need to know [ who said it, how was it said, where was it said, etc ]"
-3. Final Statement
-    - "Question Number [ 1, 2, 3, 4, 5, etc ] question"
-4. Actual Question
+A Situation question could be written:
 
-Additional rules:
+> Who said it? "Come, follow me, and I will send you out to fish for people."
 
-- For INT and MA, begin reading the question
-- For Chapter Reference (and MACR), "According to [ Book Name ], Chapter [Chapter Number]..."
-- For Chapter Verse Reference (and MACVR, "According to [ Book Name ], Chapter [Chapter Number], Verse [ Verse Number]..."
-- For Quote Questions, "Quote [ Book Name ], Chapter [Chapter Number], Verse [ Verse Number]"
-- For Quote these Two Verses Questions. "Quote [ Book Name ], Chapter [Chapter Number], Verses [ Verse Number 1] and [Verse Number 2]"
-- For all Finish Questions (and sub-types), begin reading the verse(s)
-- For Situation Questions, begin reading at the Quotation.
+Answer: "Jesus."
 
 ### Invalid Questions
 
-Invalid Questions - Questions must be declared invalid by the quizmaster or answer judge(s) if:
+Questions must be declared invalid by the quizmaster or answer judge(s) if:
 
-- The answer requires interpretation by the quizzer.
-- The question is ambiguous, tricky, or misleading.
-- The question is multiple-choice.
-- If the question is not answered.
+- The answer requires interpretation by the quizzer
+- The question is ambiguous, tricky, or misleading
+- The question is multiple-choice
+- If the question is not answered
 
-When a question is deemed invalid and thrown out, it must be replaced with the exact same type of question. (i.e. Interrogative replaced with Interrogative, MACVR replaced with MACVR, etc.)
+When a question is deemed invalid and thrown out, it must be replaced with the exact same type of question. For example, an MACVR must be replaced with an MACVR, not any multiple-answer reference question.
```

Comment:
should any use of "for example" indicate we need to move it under an Examples header?

Reply:
Only for examples that are not to be inferred as rules. A "for example" inline on the text provides an example that should be interpreted as a rule. Long-term, my preference would be to eliminate all examples that aren't in example sections, replacing them with such text that they can be understood without an inline example. But I figure that's very v-next sort of scope.